### ISSUE TITLE
Nested assignments usage

### ISSUE THREAD

**Issue by mambax7** [kalessil/phpinspectionsea#1660]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate), 3.0.10 |
| Language level | PHP 8.0 RC-4|

Current behaviour
--
I had this code:
```php
$profile_groups[] = $user_groups[] = 0;
```
After running the QuickFix of Nested assignments usage, I've got:
```php
$user_groups[]    = 0;
$profile_groups[] = $user_groups[];
```
And I've got an error in PHP8:
`Cannot use [] for reading`

Expected behaviour
```php
$user_groups[]    = 0;
$profile_groups[] = 0;
```

Environment details
PhpStorm 2020.3 EAP
Build #PS-203.5784.36, built on November 12, 2020
Runtime version: 11.0.9+11-b1145.10 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 15360M
Non-Bundled Plugins: com.kalessil.idea.jsInspectionsUltimate,

**Comment by ea-inspections-team** [comment]
Thank you for reporting @mambax7! Fixed!

**Comment by mambax7** [comment]
Thank you!!!!
